nxc/netexec.py

```diff
@@ -79,6 +79,7 @@ def main():
     else:
         nxc_logger.logger.setLevel(logging.ERROR)
         root_logger.setLevel(logging.ERROR)
+    logging.getLogger("neo4j").setLevel(logging.ERROR)
```

Comment:
Why set this to ERROR outside of the if checks?

Reply:
I disabled that because it spams the hell out of it with bloodhound enabled. Also as we make our own debug statements inside of the connector i figured we don't need their debug info anyway.

For demonstration this is before (Yeah i needed 2 screenshots because it didnt fit on one screen lol):
![image](https://github.com/Pennyw0rth/NetExec/assets/61382599/84e25e62-17a1-4cc2-bed3-75f91cee8a3a)
![image](https://github.com/Pennyw0rth/NetExec/assets/61382599/f4bc33cc-5631-43a0-8354-ab021695a973)

After: 
![image](https://github.com/Pennyw0rth/NetExec/assets/61382599/07edfc98-172a-449d-8cc0-0509d28abe8d)

Reply:
Got it! Makes sense!